src/components/Filters/index.tsx

```diff
@@ -89,23 +89,23 @@ class Filters extends React.Component<FiltersProps, FiltersState> {
 		this.state.edit.push(this.getCleanEditModel());
 	}
 
-	componentWillReceiveProps(nextProps: FiltersProps) {
+	componentDidUpdate(prevProps: FiltersProps) {
```

Comment:
Can you change this to use `getDerivedStateFromProps`? `componentDidUpdate` triggers after a render, so if state is set from this lifecycle hook, then we end up triggering another render unnecessarily.